cli/command_repository_set_parameters.go

```diff
@@ -50,6 +55,12 @@ func (c *commandRepositorySetParameters) setup(svc appServices, parent commandPa
 	cmd.Flag("epoch-delete-parallelism", "Epoch delete parallelism").IntVar(&c.epochDeleteParallelism)
 	cmd.Flag("epoch-checkpoint-frequency", "Checkpoint frequency").IntVar(&c.epochCheckpointFrequency)
 
+	if svc.enableTestOnlyFlags() {
+		cmd.Flag("add-required-feature", "Add required feature which must be present to open the repository").Hidden().StringVar(&c.addRequiredFeature)
+		cmd.Flag("remove-required-feature", "Remove required feature").Hidden().StringVar(&c.removeRequiredFeature)
```

Comment:
How about `set-features <CSV-OF-FEATURES` versus `add/remove` ? `set` could be a simpler and less confusing CLI.

Reply:
having a file introduces all kinds of problems, add/remove is a pattern we have everywhere else.

Reply:
Was proposing comma separated value/string, doesn't have to be a file though. But this is not a deal breaker.